app/src/main/kotlin/com/wire/android/ui/calling/incoming/IncomingCallViewModel.kt

```diff
@@ -126,6 +128,6 @@ class IncomingCallViewModel @Inject constructor(
     companion object {
         private const val DELAY_TIME_AFTER_ENDING_CALL = 1000L
         private val ACCEPT_CALL_DEFAULT_BACKSTACK_NODE = BackStackMode.REMOVE_CURRENT
-        private const val ACCEPT_CALL_DEFAULT_DELAY_TIME = 0L
+        private const val ACCEPT_CALL_DEFAULT_DELAY_TIME = 200L
```

Comment:
Why the increase in delay?

Reply:
Creating call metadata is a heavy task and it takes some time to be applied (we need to improve it). until that time I am adding some delay